EasyPost/IResource.cs

```diff
@@ -4,7 +4,7 @@ namespace EasyPost
 {
     public interface IResource
     {
-        Dictionary<string, object> AsDictionary();
+        Dictionary<string, object?> AsDictionary();
```

Comment:
What's with the proliferation of all these question marks throughout this PR? I thought this was only to make things async?

Reply:
The `?`  in C# means nullable. The IDE is constantly complaining that we are setting `null` as a default value to non-nullable values. So now, the values are nullable. More explicit to the end user, less likely to throw a runtime error.